src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfiguration.swift

```diff
@@ -28,7 +28,7 @@ public struct CCLConfiguration: Codable {
 			case jfnDescriptors = "JfnDescriptors"
 		}
 		
-		let jfnDescriptors: [JsonFunctionDefinition]
+		let jfnDescriptors: [CCLFunctionDescriptor]
```

Comment:
It would also be possible to use `JsonFunctionDescriptor`

Reply:
I'll fix that.